apps/web/src/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { CheckIcon } from "@heroicons/react/24/outline";
+
+import Copy from "~/icons/copy.svg";
+import { Tooltip } from "./Tooltip";
+
+export function CopyToClipboard({
+  label,
+  value,
+}: {
+  label: string;
+  value: string;
+}) {
+  const [isCopied, setCopied] = useState(false);
+  const [isHover, setIsHover] = useState(false);
```

Comment:
Can we use the `useHover` hook that we already have here?

Reply:
Done
[46f5191](https://github.com/Blobscan/blobscan/pull/478/commits/46f51912fe4690729298ee7ddc4e3ad81168c460)